ButtonStateless/index.jsx

```diff
@@ -51,6 +51,7 @@ const Button = ({
       fontFamily,
       fontSize: fontSizeSmall,
       fontWeight,
+      lineHeight: '0',
```

Comment:
For values that don't have `rem` or `px` I think an integer is the way to go here.